The APF51Dev is a docking board for an APF51 SOM

Signed-off-by: Gwenhael Goavec-Merou <gwenhael.goavec-merou@armadeus.com>
Signed-off-by: Laurent Cans <laurent.cans@gmail.com>
---
Index: linux-3.10/arch/arm/boot/dts/imx51-apf51dev.dts
===================================================================
--- linux-3.10.orig/arch/arm/boot/dts/imx51-apf51dev.dts
+++ linux-3.10/arch/arm/boot/dts/imx51-apf51dev.dts
@@ -34,6 +34,12 @@
 			gpios = <&gpio1 2 0>;
 			linux,default-trigger = "heartbeat";
 		};
+
+		backlight {
+			label = "Backlight";
+			gpios = <&gpio3 4 0>;
+			default-state = "on";
+		};
 	};
 };
 
@@ -51,6 +57,14 @@
 	fsl,spi-num-chipselects = <2>;
 	cs-gpios = <&gpio3 28 1>, <&gpio3 27 1>;
 	status = "okay";
+
+	spidev: spidev@1 {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "spidev";
+		spi-max-frequency = <25000000>;
+		reg = <1>;
+	};
 };
 
 &esdhc1 {
@@ -65,6 +79,7 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_esdhc2_1>;
 	bus-width = <4>;
+	fsl,no_sdio_irq;
 	non-removable;
 	status = "okay";
 };
@@ -91,6 +106,7 @@
 				MX51_PAD_CSPI1_SS1__GPIO4_25 0x0C5
 				MX51_PAD_GPIO1_2__GPIO1_2    0x0C5
 				MX51_PAD_GPIO1_3__GPIO1_3    0x0C5
+				MX51_PAD_DI1_D1_CS__GPIO3_4  0x0C5
 			>;
 		};
 	};
